exercises/G-numbers/exercise.js

```diff
@@ -1 +1,5 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let numOfStudents=15;
+let numOfMentors=8;
```

Comment:
Same issue again with spaces. Make sure you come back and tweak these lines :smiley: 
```suggestion
let numOfMentors = 8;
```